src/app/modules/authoring/learning-tasks/domain-model/domain-model.component.html

```diff
@@ -0,0 +1,51 @@
+<div fxLayout="column">
+    <mat-card fxFlex fxLayout="column" fxLayoutGap="20px" class="sticky">
```

Comment:
Lines 2-28 are copy-pasted from case-studies, along with the related SCSS and TypeScript code. What would be a better solution to reduce redundancy?

Reply:
To create new component for this part?